app/views/operational_fields/show.html.erb

```diff
@@ -21,7 +21,6 @@
     </div>
   </div>
 </header>
-
```

Comment:
Non-blocker: Are all these space removals coming from a linter in your editor or something? It's unnecessary to remove them as rails compiles the html on build, these are more for readability.

Reply:
Yeah exactly - part of a beautifier to automate formatting. I've restored, empty line settings need a tweak.